<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Userlten</class>
 <widget class="QDialog" name="Userlten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>57</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>278</width>
    <height>57</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>278</width>
    <height>57</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="bp_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="ib_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>53</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>小饼夹肉</string>
   </property>
  </widget>
  <widget class="QLabel" name="ib_feeling">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Excalibur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
